src/DashboardUI/src/pages/admin/AdminOrganizationUsersPage.tsx

```diff
@@ -61,23 +67,86 @@ export function AdminOrganizationsUsersPage() {
     }
   };
 
+  const openRemoveUserModal = (userId: string) => {
+    setShowRemoveUserModal(!showRemoveUserModal);
+    setRemoveUserId(userId);
+  };
+
+  const closeRemoveUserModal = () => {
+    setShowRemoveUserModal(false);
+    setRemoveUserId('');
+  };
+
   return (
     <>
       <div className="overflow-y-auto h-100">
         <div className="m-3">
           {pageTitle()}
 
-          <div className="d-flex justify-content-between my-4">
-            <h2>Users</h2>
+          <div className="d-flex justify-content-between align-items-center my-4">
+            <h2 className="fs-5">Users</h2>
             {addUserButton()}
           </div>
 
           <TableLoading
             isLoading={organizationListLoading || organizationUsersListLoading}
             isErrored={organizationListErrored || organizationUsersListErrored}
           >
-            (<pre>OrganizationUsers: {JSON.stringify(organizationUsersListResponse)}</pre>)
+            <table className="w-100">
```

Comment:
Any reason you didn't use the bootstrap table here? It might give you a lot of the border and spacing that you added in by hand

```suggestion
            <table className="table">
```